addons/gdUnit4/src/network/GdUnitTcpServer.gd

```diff
@@ -8,112 +8,155 @@ signal rpc_data(rpc_data)
 
 var _server :TCPServer
 
+
 class TcpConnection extends Node:
 	var _id :int
-	var _stream : StreamPeerTCP
+	var _stream
```

Comment:
We need to remove the type otherwise the test is failing because it can hold a mocked stream.
With Godot4 are not allowed to override core functions!